pkg/config/config.go

```diff
@@ -144,6 +144,7 @@ var defaultServerConfig = &ServerConfig{
 	LogFile:       "",
 	LogLevel:      "info",
 	GcTTL:         24 * 60 * 60, // 24H
+	GcBLT:         24 * 60 * 60, // 24H
```

Comment:
Should we just reuse the `GcTTL` parameter? If we reuse it, the GC safe time delay thresholds will be uniform regardless of whether the CDC server is running or not.

Reply:
@amyangfei @liuzix

Reply:
I think it is a good idea.

Reply:
Agreed